File: Week 1 - C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);
void print_blocks(int size);

int main(void)
{
    // Get number from user. Has to be between 1 and 8.
    int n = get_height();

    // Print blocks based on height
    print_blocks(n);
}

int get_height(void)
{
    int n;

    do
    {
        n = get_int("Size of tower: ");
    }
    while (n < 1 || n > 8);

    return n;
}

void print_blocks(int size)
{
    int space = size;
    int hashtag = 1;

    for (int j = 0; j < size; j++)
    {

        // Print the number of spaces
        for (int i = 0; i < (space - 1); i++)
        {
            printf(" ");
        }

        // Print the number of hashtags on left side
        for (int i = 0; i < hashtag; i++)
        {
            printf("#");
        }
        // Print the two spaces in the hole
        printf("  ");

        // Print the number of hashtags on right side
        for (int i = 0; i < hashtag; i++)
        {
            printf("#");
        }

        space--;
        hashtag++;
        printf("\n");
    }
}

File: Week 6 - Python/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    n = get_height()
    print_blocks(n)


def get_height():
    while True:
        n = get_int("Height: ")
        if n < 9 and n > 0:
            return n


def print_blocks(size):
    space = size - 1
    hashtag = 1

    while hashtag <= size:
        print(" " * space + "#" * hashtag + " " * 2 + "#" * hashtag)
        hashtag += 1
        space = space - 1


main()
